src/Explorer/Controls/NotebookGallery/InfoComponent/InfoComponent.tsx

```diff
@@ -25,6 +29,11 @@ export class InfoComponent extends React.Component {
         <Stack.Item>
           {this.getInfoPanel("KnowledgeArticle", "Microsoft Terms of Use", CodeOfConductEndpoints.termsOfUse)}
         </Stack.Item>
+        {this.props.onReportAbuseClick !== undefined && (
```

Comment:
Would use !this.props.onReportAbuseClick so we also cover nulls

Reply:
Addressing in my next PR